bullet_train-api/bullet_train-api.gemspec

```
@@ -32,7 +32,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "pagy_cursor"
   spec.add_dependency "rack-cors"
   spec.add_dependency "doorkeeper"
-  spec.add_dependency "jbuilder-schema", "2.6.2"
+  spec.add_dependency "jbuilder-schema", "2.6.4"
```

Now that we can [test `jbuilder-schema` against the starter repo](https://github.com/bullet-train-co/jbuilder-schema/pull/73) prior to releasing a new version I think we should try allowing the patch number to float again.
```suggestion
  spec.add_dependency "jbuilder-schema", "~> 2.6.4"
```